README.md

```diff
@@ -221,6 +221,17 @@ With some additional manual work the upstream upgrade tool will update your appl
 
 (Just add an additional `.` to all the paths referenced)
 
+**Update** references to any existing css files imported in the `app/assets/stylesheets/application.tailwind.css`
```

Comment:
This addition is in the "Updating CSS class names for v4" section, which I don't think is right.

I'd prefer it to be in the earlier "Upgrade steps" section. I'd also like to keep it short to keep that section scannable (it's already a bit long), can you shorten the diff?

Reply:
Done. You can preview it here: https://github.com/frayzil/tailwindcss-rails?tab=readme-ov-file#upgrade-steps

Please let me know if you want any more changes.